### PR TITLE
[W6.1e][W11-B3] Nguyen Thi Viet Ha

### PR DESCRIPTION
**Pull request by deeheenguyen**
add test to isAnyNull()

### CODE REVIEW

**Comment by okkhoy** [comment]
@deeheenguyen Please update the title of the PR to reflect your phase B team name.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -36,11 +36,45 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception{
```
Comment (joanneong) [inline test/java/seedu/addressbook/common/UtilsTest.java:40]: Can have a spacing between `Exception` and `{`

**Review by joanneong — COMMENTED** [review]
In general, this is a good attempt made :)

Just a reminder that in general, test names follow a 3-part naming rule: FeatureUnderTest_scenarioBeingTested_expectedOutcome(), for example: isAnyNull_nullList_returnsTrue().

However, in this case, it's also fine to omit the second and third parts (like you have done) and then comment inside the test case itself :) Note though that this makes the test 'broader', so if one small case fails inside the entire test case will fail, so sometimes people prefer to do the three-part-naming way.

Lastly, although your current test divisions are okay, you can also consider testing cases like: (1) empty list, (2) any non-empty list, (3) non-empty list with just one null at the beginning, (4) non-empty lists with nulls in the middle, (5) non-empty list with one null as the last element etc.
